src/page-multi-facility/ReferenceDataModal/context/ReferenceDataModalContext.tsx

```diff
@@ -205,18 +231,7 @@ export const ReferenceDataModalProvider: React.FC<{ syncType: SyncType }> = ({
               },
             });
 
-            const scenarioChange: Partial<Scenario> = {
-              referenceDataObservedAt: new Date(),
-            };
-            // only want to show this once as a promo;
-            // closing the modal should dismiss the promo status
-            if (scenario?.promoStatuses.referenceData) {
-              scenarioChange.promoStatuses = {
-                ...scenario.promoStatuses,
-                referenceData: false,
-              };
-            }
-            handleScenarioChange(scenarioChange);
+            clearPromoStatus;
```

Comment:
Should this not be `clearPromoStatus();`? Or do I misunderstand?

Reply:
wow whoops 😎